### PR TITLE
Use GitHub actions to check, lint, build and release.

### PR DESCRIPTION
**Pull request by gwarf**
Use GitHub actions to check, lint, build and release.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            build/RPMS/noarch/bdii-config-site-*.el7.noarch.rpm
+
+  # Use a matrix for CentOS Stream versions
+  build:
+    strategy:
+      matrix:
+        centos-version: [8, 9]
+    name: Build CentOS ${{ matrix.centos-version }} RPMs
+    runs-on: ubuntu-latest
+    container: quay.io/centos/centos:stream${{ matrix.centos-version }}
```
Comment (andrea-manzi) [inline .github/workflows/build.yml:39]: maybe we should use almalinux or rockylinux instead ? i think nothing will break but those are the OS to be supported now

Comment (gwarf) [reply .github/workflows/build.yml:39]: Until now I just (lazily...) followed with what I did for other packages, building on CentOS Stream, especially as we don't expect much impact/differences especially for the python / shell stuff.
But I agree switching is probably best. I was thinking of switching to Alma at some point, as it seems it's expected to be the one that will be selected / "official" in the end.
So let's try to start building on Alma this package.

Comment (gwarf) [reply .github/workflows/build.yml:39]: Will try using their official containers: https://wiki.almalinux.org/containers/docker-images.html 

Will do it for both 8 and 9. Will be simpler than keeping something on CentOS Stream for 8 and only move Alma to 8, and I guess it should be OK.

Comment (andrea-manzi) [reply .github/workflows/build.yml]: thanks!

**Review by andrea-manzi — APPROVED** [review]
LGTM! let's try this new almalinux;-)

**Pull request merged**